src/Visualizer/Visualizer.js

```diff
@@ -32,21 +40,22 @@ class Visualizer {
   change_node_position(new_position) {
     let Node = new_position.node;
     Node.position = new_position.position;
+    new_position.animate();
   }
-
+  // new node color will by only chaned in the Graph  and won't be animated
```

Comment:
I agree for now...but maybe in the future, we can create a good animation for it?

Reply:
Right now I don't have any idea why and how to animate the color of the node, so we can leave it for now until we have a reason to animate the node color.